src/components/pages/roster-page/roster-page.tsx

```diff
@@ -99,11 +100,17 @@ export const RosterPage = () => {
   };
 
   function initializeTable() {
+
+    dispatch(AppFrame.setPageLoading(true));
+
     fetch(`api/roster/${orgId}/count`).then(async response => {
       const countResponse = (await response.json()) as CountResponse;
       setRosterSize(countResponse.count);
       await handleChangePage(null, 0);
     });
+
+    dispatch(AppFrame.setPageLoading(false));
```

Comment:
This function needs to be updated to be async and instead use `axios.get`.  As is, the page loading will be set to `false` before the asynchronous API request completes.

Reply:
Good catch. Updated and tested this. Should be working correctly now with an axios.get.